extension/scripts/initApiExtensions.js

```diff
@@ -0,0 +1,19 @@
+const Promise = require('bluebird')
+const _ = require('lodash')
+
+const apiExtensionDrafts = require('../resources/api-extensions.json')
+
+async function initApiExtensions (ctpClient, ctpAdyenIntegrationBaseUrl) {
+  await Promise.map(apiExtensionDrafts.values(), async (uncompiledDraft) => {
+    try {
+      const extension = _.template(JSON.stringify(uncompiledDraft))({ ctpAdyenIntegrationBaseUrl })
+      await ctpClient.create(ctpClient.builder.extensions, extension)
+    } catch (e) {
+      console.log('Error when creating interface interaction custom type, skipping...', JSON.stringify(e))
```

Comment:
```suggestion
      console.log('Error when creating API extension, skipping...', JSON.stringify(e))
```

Reply:
I think it should be logged as an error. But here you don't have a logger. So I'd write it as a `console.error()` for now.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/4/commits/973138fc68d46f5c3c1cdd735c0f7e8932d55bc2